lib/diagnostic.js

```diff
@@ -16,6 +16,7 @@ db.dropDatabase();
 
 // your code begins here
 
+db.createCollection('houses', { name: 'Arryn', motto: 'As High as Honor'  }, { house: 'stark',  motto: 'Winter is Coming' }, { house: 'Targaryen', motto: 'Fire and Blood' } );
```

Comment:
You can actually create the collection and its documents all at once without explicitly creating the collection first.  Here are two possible solutions.

```javascript
db.houses.insert({
  name: 'House Arryn',
  motto: 'As High as Honor'
});

db.houses.insert({
  name: 'House Stark',
  motto: 'Winter is Coming'
});

db.houses.insert({
  name: 'House Targaryen',
  motto: 'Fire and Blood'
});
```

or

```javascript
db.houses.insert([{
  name: 'House Arryn',
  motto: 'As High as Honor'
}, {
  name: 'House Stark',
  motto: 'Winter is Coming'
}, {
  name: 'House Targaryen',
  motto: 'Fire and Blood'
}]);
```